app/src/main/java/com/android/calendar/ImportActivity.java

```diff
@@ -181,9 +181,13 @@ private void parseCalFile() {
 
         String dtEnd = firstEvent.getProperty(VEvent.DTEND);
         String dtEndParam = firstEvent.getPropertyParameters(VEvent.DTEND);
-        if (!TextUtils.isEmpty(dtEnd)) {
+        if (dtEnd != null && !TextUtils.isEmpty(dtEnd)) {
```

Comment:
Nit: `null` check is redundant, https://developer.android.com/reference/android/text/TextUtils#isEmpty(java.lang.CharSequence) checks for it already.